src/rpc/message.h

```diff
@@ -28,27 +28,12 @@
 
 #pragma once
 
-#include "rapidjson/document.h"
-#include "rpc/message_data_structs.h"
+#include <rapidjson/document.h>
```

Comment:
Does this still find the one in our tree ?

Reply:
Yes. I used the brackets because its not in our codebase or maintained by us, but perhaps that styling was too aggressive ?

Reply:
I don't care as long as it doesn't break that case.